### PR TITLE
Language Metrics

### PR DESCRIPTION
**Pull request by MatthewSchultz**
Shows language metrics

[tracker](https://www.pivotaltracker.com/story/show/173270982)

### CODE REVIEW

**Comment by MatthewSchultz** [comment]
The hakiri thing is already fixed

File: app/helpers/metrics_report.rb [hunk 1]
```diff
@@ -27,6 +27,24 @@ def median_ttc
...
+  def submitted_language_percents
+    count_by_language = Household.where.not(submitted_at: nil).group(:language).count
+    total = count_by_language.values.sum.to_f
+    count_by_language.map { |lang, count| [lang, ((count.to_f / total) * 100).round(2)] }.to_h
+  end
+
+  def unsubmitted_language_percents
+    count_by_language = Household.where(submitted_at: nil).group(:language).count
+    total = count_by_language.values.sum.to_f
+    count_by_language.map { |lang, count| [lang, ((count.to_f / total) * 100).round(2)] }.to_h
```
Comment (CoryFoy) [inline app/helpers/metrics_report.rb:45]: We could probably refactor these out, but not critical

File: app/helpers/ordinal_columns_helper.rb [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    when 4
+      'width-one-fourth'
+    when 5
+      'width-one-fifth'
+    when 6
+      'width-one-sixth'
+    when 7
+      'width-one-seventh'
+    when 8
+      'width-one-eigth'
```
Comment (CoryFoy) [inline app/helpers/ordinal_columns_helper.rb:19]: Does this need to be `eighth`?

Comment (MatthewSchultz) [reply app/helpers/ordinal_columns_helper.rb:19]: it does, yes. Will fix. I'm decent at programming, less good at spelling

**Pull request merged**